tests/test_cli_utils.py

```diff
@@ -11,6 +12,235 @@
         ("samples/pipeline_template_examples/testing/example.yaml", False),
     ],
 )
-def test_is_sweep_pipeline(pipeline_file: Path, expected_is_sweep_pipeline: bool):
+def test_is_sweep_pipeline_file(pipeline_file: Path, expected_is_sweep_pipeline: bool):
+    """Test is_sweep_pipeline with file paths"""
     pipeline_file_path = Path(__file__).parent / pipeline_file
     assert is_sweep_pipeline(pipeline_file_path) is expected_is_sweep_pipeline
+
+
+def test_is_sweep_pipeline_dict_with_range():
+    """Test is_sweep_pipeline with a Python dict containing a sweep range"""
```

Comment:
Is this docstring correct, it mentions a python dict, but this looks to be a string containing JSON, rather than a python dict?